Source/FakeItEasy.Net35/Compatibility.cs

```diff
@@ -51,28 +51,7 @@ where entry.Item1.IsAlive
     }
 }
 
-namespace System.ComponentModel.Composition
-{
-    using System.Diagnostics.CodeAnalysis;
-
-    [SuppressMessage("Microsoft.Performance", "CA1813:AvoidUnsealedAttributes", Justification = "Mimicks net40 BCL type.")]
-    [SuppressMessage("StyleCop.CSharp.MaintainabilityRules", "SA1402:FileMayOnlyContainASingleClass", Justification = "Tidier.")]
-    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Interface, AllowMultiple = true, Inherited = true)]
-    public class InheritedExportAttribute
-        : Attribute
-    {
-    }
-
-    [SuppressMessage("Microsoft.Performance", "CA1813:AvoidUnsealedAttributes", Justification = "Mimicks net40 BCL type.")]
-    [SuppressMessage("StyleCop.CSharp.MaintainabilityRules", "SA1402:FileMayOnlyContainASingleClass", Justification = "Tidier.")]
-    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Field | AttributeTargets.Parameter, AllowMultiple = false, Inherited = false)]
-    public class ImportManyAttribute
-        : Attribute
-    {
-    }
-}

```

Comment:
I think that these became unnecessary in or around 2b33fa75b18950e751552f87bf03af050974ae8a.
I'm going to spin off a new issue for removal. Issue mostly because I'm _slightly_ nervous about removing any public class, and I'd like to be able to point at the change.